src/Traits/HasRoles.php

```diff
@@ -311,6 +311,16 @@ public function getAllPermissions(): Collection
             ->values();
     }
 
+    /**
+     * Return a collection of role names associated with this user.
+     *
+     * @return Collection
+     */
+    public function getRoleNames(): Collection
```

Comment:
The function signatures is clear enough and contains all info. Let's remove that docblock